test/dataTypes.spec.ts

```diff
@@ -416,3 +436,42 @@ describe('recursive typings', function () {
     assert.ok(assertType<typeof a, Buffer>(true))
   })
 })
+
+describe('getLength', function () {
+  it('should return the right RLP length', function () {
+    assert.ok((<Buffer>RLP.getLength(0)).equals(Buffer.from([])))
```

Comment:
Was confused while writing this test. Why does `getLength` return empty buffer in case that the input has zero length?

Reply:
If `getLength` is used as part of the decoding process, it seems like it would just be following the definition of rlp encoded integers such that 0 is encoded its most compact form or the empty buffer.  I looked at the line history and don't see any specific commentary on this in past PRs.

Reply:
You would think it is part of the decoding process, but actually it is not. It is a standalone method and in `index.ts` you can remove it without breaking anything. So it is an external method to "get the length of the RLP input" (what does this mean anyways? That would be just the input Buffer length..? Or do we mean the RLP data length here?), why would this return a number in most cases and a Buffer in edge cases?

Reply:
In that case, let's fix it.  This is ancient code that's due for a refresh, right?  If we're doing a breaking release, let's rationalize the API and return a 0 here for a 0 length input.

Reply:
thanks @acolytec3 and @jochem-brouwer for opening a PR, will resolve this conversation and can continue in #106